src/geometry/Polygon2d.h

```diff
@@ -40,7 +40,7 @@ class Polygon2d : public Geometry
 
   typedef std::vector<Outline2d> Outlines2d;
   const Outlines2d& outlines() const { return theoutlines; }
-
+  using Geometry::transform;
```

Comment:
This looks like an incorrect "using" include (to fix that warning), and maybe indicative of a failed is-a relationship in the inheritance if Geometry was expected to take a 3D transform (although it's not even implemented in the base).  To think about.

Reply:
I was trying to think of a nice clean solution, but haven't come up with anything yet.

But also keep in mind, that if OpenSCAD is to officially support 2d-in-3d geometry at some point, then we would probably still want some way to call a 3d transform on 2d object.  
Then again, maybe the class to do that would be something entirely separate from Polygon2d.

Reply:
added note in comments